lib/WebSocketClient.js

```diff
@@ -112,6 +112,40 @@ function WebSocketClient(config) {
 
 util.inherits(WebSocketClient, EventEmitter);
 
+function handleWsfProtocol(url) {
+  var path = require('path');
+  var fs = require('fs');
```

Comment:
I don't like at all this. Modules should not be loaded within functions but on the top of the file.